### PR TITLE
Changelog for admin-date-time package

### PR DESCRIPTION
**Pull request by jamesricky**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,11 +47,15 @@ Migrate to MUI 5, following the official [MUI Migration Guide](https://mui.com/g
...
+## @comet/admin-date-picker -> @comet/admin-date-time
+
+-   Added standalone `DatePicker` and `DateRangePicker` components for use outside of FinalForm
 
 ### Incompatible Changes
 
-The date-picker package has been removed.
+-   `@comet/admin-date-picker` has been renamed `@comet/admin-date-time`, and the date-picker components have been re-implemented from scratch.
+-   The props that previously were passed through to the [react-dates](https://github.com/react-dates/react-dates) components will no longer have any effect, as [react-date-range](https://github.com/hypeserver/react-date-range) is now used internally
+-   The types of the change-value remain the same as with the previous date-picker components
```
Comment (johnnyomair) [inline CHANGELOG.md:58]:
```suggestion
-   The types of the change value remain the same as with the previous date picker components
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,11 +47,15 @@ Migrate to MUI 5, following the official [MUI Migration Guide](https://mui.com/g
...
-## @comet/admin-date-picker
+## @comet/admin-date-picker -> @comet/admin-date-time
+
+-   Added standalone `DatePicker` and `DateRangePicker` components for use outside of FinalForm
 
 ### Incompatible Changes
 
-The date-picker package has been removed.
+-   `@comet/admin-date-picker` has been renamed `@comet/admin-date-time`, and the date-picker components have been re-implemented from scratch.
+-   The props that previously were passed through to the [react-dates](https://github.com/react-dates/react-dates) components will no longer have any effect, as [react-date-range](https://github.com/hypeserver/react-date-range) is now used internally
```
Comment (johnnyomair) [inline CHANGELOG.md:57]: Do we need this information? Seems like an implementation detail

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,11 +47,15 @@ Migrate to MUI 5, following the official [MUI Migration Guide](https://mui.com/g
...
 
-## @comet/admin-date-picker
+## @comet/admin-date-picker -> @comet/admin-date-time
+
+-   Added standalone `DatePicker` and `DateRangePicker` components for use outside of FinalForm
 
 ### Incompatible Changes
 
-The date-picker package has been removed.
+-   `@comet/admin-date-picker` has been renamed `@comet/admin-date-time`, and the date-picker components have been re-implemented from scratch.
```
[the remark this answers is not in the document]
Comment (jamesricky) [reply CHANGELOG.md:56]: According to Grammarly the `to` is unnecessary 🙂

File: CHANGELOG.md (hunk 2 above)
Comment (jamesricky) [reply CHANGELOG.md:57]: I think the package rename & the fact that some props you could add to the old picker components will no longer work are important. 

But we could probably remove this part: `the date-picker components have been re-implemented from scratch`

Comment (johnnyomair) [reply CHANGELOG.md:57]: Maybe we could rephrase it to `The date picker library used internally has been changed from `[react-dates](https://github.com/react-dates/react-dates) to [react-date-range](https://github.com/hypeserver/react-date-range). Props specific to react-dates will no longer have any effect.

Comment (johnnyomair) [reply CHANGELOG.md:57]: > But we could probably remove this part: the date-picker components have been re-implemented from scratch

Yes, we should remove that.
